server/data/data-jdbc/src/main/java/org/apache/james/mailrepository/jdbc/JDBCMailRepository.java

```diff
@@ -255,15 +247,10 @@ public void init() throws Exception {
                 sr.configure(streamConfiguration);
                 sr.init();
 
-                if (LOGGER.isDebugEnabled()) {
-                    LOGGER.debug("Got filestore for JdbcMailRepository: " + filestore);
-                }
+                LOGGER.debug("Got filestore for JdbcMailRepository: {}", filestore);
             }
 
-            if (LOGGER.isDebugEnabled()) {
-                String logBuf = this.getClass().getName() + " created according to " + destination;
-                LOGGER.debug(logBuf);
-            }
+            LOGGER.debug("{} created according to {}", getClass().getName(), destination);
         } catch (Exception e) {
             final String message = "Failed to retrieve Store component:" + e.getMessage();
             LOGGER.error(message, e);
```

Comment:
Looks like we can:
 - get ride of the getMessage
 - inline the message in the log operation

Reply:
Done